handlebars/monster_handlebar.hbs

```diff
@@ -20,77 +21,111 @@ license: {{system.details.publication.license}}
-{{#each system.traits.value as |trait index|}}
-trait_0{{add index 1}}: "{{trait}}"
+level: Creature {{system.details.level.value}}
+rarity: {{system.traits.rarity}}
+alignment: {{more-handlebars-helpers-table system.details.alignment.value "LG" "Lawful Good" "LN" "Lawful Neutral" "LE" "Lawful Evil" "NG" "Neutral Good" "N" "Neutral" "NE" "Neutral Evil" "CG" "Chaotic Good" "CN" "Chaotic Neutral" "CE" "Chaotic Evil"}}
+size: {{more-handlebars-helpers-table system.traits.size.value "ti" "Tiny" "sm" "Small" "med" "Medium" "lg" "Large" "hu" "Huge" "ga" "Gargantuan"}}
+{{#if system.traits.value}}
+traits:
+{{#each system.traits.value as |trait|}}
+  - {{trait}}
 {{/each}} 
+{{/if}}
+
 modifier: {{system.perception.value}}
-perception:
-  - name: "Perception"
-    desc: "{{numberFormat system.perception.value sign=true}}; {{#each system.perception.senses as |sense|}}{{sense.label}}{{ifThen @last "" ", "}}{{/each}}"
+{{#if system.perception.senses}}
+senses: {{#each system.perception.senses as |sense|}}{{sense.label}}{{ifThen @last "" ", "}}{{/each}}
+{{/if}}
 {{#if (or system.details.languages.value system.details.languages.details)}}
 languages: "{{#each system.details.languages.value as |language|}}{{capitalize language}}{{ifThen @last "" ", "}}{{/each}}{{#if system.details.languages.details}}{{#if system.details.languages.value}}; {{/if}}{{system.details.languages.details}}{{/if}}"
 {{/if}}
+{{#if (me-getItemsByType items "lore")}}
 skills:
-  - name: "Skills"
-    desc: "{{#each items}}{{#if (eq type "lore")}}{{{name}}}: {{numberFormat system.mod.value sign=true}}{{#if system.variants.0.label}} ({{system.variants.0.label}}){{/if}}{{ifThen @last "" ", "}}{{/if}}{{/each}}"
+{{#each (me-getItemsByType items "lore")}}
+  - "{{{name}}}": {{#if system.variants.0.label}}"{{system.mod.value}} ({{system.variants.0.label}})"{{else}}{{system.mod.value}}{{/if}}
+{{/each}}
+{{/if}}
 abilityMods: [{{system.abilities.str.mod}}, {{system.abilities.dex.mod}}, {{system.abilities.con.mod}}, {{system.abilities.int.mod}}, {{system.abilities.wis.mod}}, {{system.abilities.cha.mod}}]
 speed: {{#if system.attributes.speed.value}}{{system.attributes.speed.value}} feet{{#if system.attributes.speed.otherSpeeds}}, {{/if}}{{/if}}{{#each system.attributes.speed.otherSpeeds}} {{type}} {{value}} feet{{ifThen @last "" ", "}}{{/each}}
-sourcebook: "_{{system.details.publication.title}}_"
+
 ac: {{system.attributes.ac.value}}
-armorclass:
-  - name: AC
-    desc: "{{system.attributes.ac.value}}{{#if system.attributes.ac.details}} {{system.attributes.ac.details}}{{/if}}; __Fort__ {{numberFormat system.saves.fortitude.value sign=true}}, __Ref__ {{numberFormat system.saves.reflex.value sign=true}}, __Will__ {{numberFormat system.saves.will.value sign=true}}{{#if system.attributes.allSaves.value}}; {{system.attributes.allSaves.value}}{{/if}}"
+{{#if system.attributes.ac.details}}
+acNote: {{system.attributes.ac.details}}
+{{/if}}
+saves:

```

Comment:
I haven't had a chance to install the new statblock format.  How do the saves end up getting rendered with this new format?   My original goal was to try to mimic the look of the printed statblocks in the Monster Core, which have the AC followed by the saves all in the same line.   I can see the value in have individually tagged YAML elements if one wants to later post-process the data to filter things or extra fields to pull into another note, but will it still look the same visually when rendered?

Maybe I'm being too obsessed with trying to mimic the look of the printed statblocks?  For example, one is from the book and the other is from Obsidian:
![Screenshot 2024-07-15 at 12 34 17 PM](https://github.com/user-attachments/assets/b8b5f8bc-0b67-4c1c-b991-40334e131414)
<img width="432" alt="Screenshot 2024-07-15 at 12 33 17 PM" src="https://github.com/user-attachments/assets/49cc590b-7e4e-4f6a-bedc-3ca0b5d17c27">

Reply:
I found an example rendering you provided here:  https://github.com/Obsidian-TTRPG-Community/Fantasy-Statblocks-CSS-Development/pull/11
That looks great to me and alleviates my main concern!